src/main/kotlin/app/extractors/PLpgSQLExtractor.kt

```diff
@@ -0,0 +1,11 @@
+// Copyright 2018 Sourcerer Inc. All Rights Reserved.
+// Author: Alexander Surkov (alex@sourcerer.io)
+
+package app.extractors
+
+object PLpgSQLExtractor : ExtractorBase(
```

Comment:
Can we add PLpgSQL with #323?

Reply:
sure, this branch is based on on that branch, as long as that thing is landed, this one will be rebased relative master, thus no merging and stuff